src/components/Navbar.tsx

```
@@ -0,0 +1,113 @@
+              <Button variant="custom" size="lg" type="submit">
+                Login With Github
+              </Button>
+            </Form>
+          )}
+        </div>
+
+        {/*  Github auth finished here  */}
+        {/*  Google auth started here  */}
+
+        <div className="flex items-center gap-5">
+          {session && session?.user ? (
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)